cpp/ps0003/problem 3 p301F Salwa واجب الحساب 2.cpp

```diff
@@ -4,27 +4,21 @@ using namespace std;
 
 int main()
 {
-cout<<"--------------------------------------------" <<endl;
-cout<<"Problem 0003"<< endl;
-
+    // prompting user for number 2/2
     int num;
-
-
-cout<<"enter the number" << endl;
-cin>> num;
-
-if (num % 2==0)
-{
-    cout <<"your number is even \n1";
-}
-else
-{
-    cout <<"your number is odd \n1";
-
-}
-
-
-
-
+    
+    cout << "enter the number" << endl;
+    cin >> num;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number